### ISSUE TITLE
AttributeError with numeric key in TaupageConfig

### ISSUE THREAD

**Issue by mfellner** [zalando-stups/senza#125]
Previously, it was possible to have a configuration like this:

```yaml
- AppServer:
    Type: Senza::TaupageAutoScalingGroup
    TaupageConfig:
      ports:
        8181: 8181
```

But the key of the port value causes an error now, unless it is surrounded in `"`, i.e. `"8181": 8181`

```
File "/usr/local/lib/python3.5/site-packages/senza/components/taupage_auto_scaling_group.py", line 98, in transform
    if len(node) == 1 and (key == "Ref" or key.startswith("Fn::")):
AttributeError: 'int' object has no attribute 'startswith'
```

Looks like `key` is not a string at https://github.com/zalando-stups/senza/blob/master/senza/components/taupage_auto_scaling_group.py#L98.

**Comment by chris-zen** [comment]
The fix for this bug is ready here #124 
Until it is not merged the solution is to downgrade to 0.82

**Comment by hjacobs** [comment]
This bug was fixed by @chris-zen --- works in the latest Senza release.
